src/dmd/dimport.d

```diff
@@ -120,18 +120,24 @@ extern (C++) final class Import : Dsymbol
         return si;
     }
 
-    void load(Scope* sc)
+    /*******************************
+     * Load this module.
+     * Returns:
+     *  true for errors, false for success
```

Comment:
`true` for error is quite counter-intuitive.

Reply:
Normally 0 (false) is success, and !=0 (true) for errors, because of returning error codes.

Reply:
This is not returning an error code so these conventions don't apply 😉.